File: level 66/classwork/task 2.js
//დავალება: for-ლუპის დახმარებით ჩაწერეთ ლისტში 1-დან 30-მდე რიცხვები, და შემდეგ შექმენით ახალი ლისტი სადაც ჩაწერთ იმ ლისტში მყოფ რიცხვებიდან მარტო ლუწ რიცხვებს

for (let i = 1; i < 30; i++){
    list1 = i;
    list2 = [];
    if (i % 2 == 1){
        list2.push(i)
    }
    console.log(list2)
}

File: level 34/homework/task 2.py
def find_short(s):
    #we just made a function
    list1 = s.split(" ")
    #we just spit it into a list for every space in the sentance

    word_len = len(list1[0])
    #we  made a variable word_len and we saved 0 in it
    for i in list1:
    #we just looked over it
        if len(i) < word_len:
        #we said that if len(i) is not greater then 0...
            word_len = len(i)
            #then word_len = number that i is in the list
    
    
    return word_len
    #we just returned it

print(find_short("bitcoin take over the world maybe who knows perhaps"))
#and finally we called it so it would have something in terminal

#done